src/lib/MessageContainer.svelte

```diff
@@ -14,28 +23,42 @@
     export const getLast = () => messages[messages.length - 1];
 
     export const addMessage = (message: MessageType) => {
-        messages.push(message);
-        messages = messages;
+        addMessageToStore(message);
+        // messages.push(message);
```

Comment:
Once I approve this PR, can you remove all of these commented out lines? You can refer to this as `dead code` which we'll try to avoid

Reply:
Yep! I usually like to keep the big old code there just in case if we decided to not go with the new changes. Will remove them after merging!

Reply:
Sweet! Let's do it actually before you merge (but it can be after I approve -- so final step)

Reply:
ok sounds good!